librz/core/canalysis.c

```diff
@@ -6726,4 +6726,130 @@ RZ_API bool rz_core_analysis_esil_trace_stop(RzCore *core) {
 	esil->trace = NULL;
 	rz_config_set_i(core->config, "dbg.trace", false);
 	return true;
-}
\ No newline at end of file
+}
+
+/**
+ * Free RzAnalysisBytes
+ *
+ * @param ptr RzAnalysisBytes pointer
+ */
+RZ_API void rz_analysis_bytes_free(void *ptr) {
+	rz_return_if_fail(ptr);
```

Comment:
free methods must accept NULL PTR (also do not forget to add NULLABLE before the `void *ptr`)